web/client/utils/__tests__/LocaleUtils-test.js

```diff
@@ -80,4 +80,17 @@ describe('LocaleUtils', () => {
         expect(LocaleUtils.getLocalizedProp('it-IT', localizedObjectProp)).toBe('titolo');
         expect(LocaleUtils.getLocalizedProp('fr-FR', localizedObjectProp)).toBe('title');
     });
+    it('getDateFormat', () => {
+        expect(LocaleUtils.getDateFormat()).toBe("YYYY/MM/DD");
```

Comment:
I suppose this is a check for unknown locale. Please describe in a comment

Reply:
for getting the default one